bundles/org.dataflowanalysis.converter/src/org/dataflowanalysis/converter/interactive/ConvertTask.java

```diff
@@ -0,0 +1,208 @@
+package org.dataflowanalysis.converter.interactive;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.Scanner;
+import org.apache.log4j.Logger;
+import org.dataflowanalysis.converter.Converter;
+import org.dataflowanalysis.converter.ConverterModel;
+import org.dataflowanalysis.converter.ModelType;
+import org.dataflowanalysis.converter.PersistableConverterModel;
+import org.dataflowanalysis.converter.dfd2web.DataFlowDiagramAndDictionary;
+import org.dataflowanalysis.converter.micro2dfd.MicroConverterModel;
+import org.dataflowanalysis.converter.pcm2dfd.PCMConverterModel;
+import org.dataflowanalysis.converter.plant2micro.PlantConverterModel;
+import org.dataflowanalysis.converter.web2dfd.WebEditorConverterModel;
+
+public class ConvertTask {
```

Comment:
As this is the class that is executed as java application there is probably a better name, maybe something in the direction of interactive, standalone, ...?

Reply:
Renamed to `StandaloneConvertTask` in 4dfe3cb5002d